src/phone-input/constants.js

```diff
@@ -11,6 +11,7 @@ export const DEFAULT_MAX_DROPDOWN_HEIGHT = '400px';
 export const STATE_CHANGE_TYPE = {
   textChange: 'textChange',
   countryChange: 'countryChange',
+  clear: 'clear',
```

Comment:
more abstractly, this overlaps the `textChange` event right? do consumers need to have a more refined change type? What use-case do we expect?

Reply:
i guess this can just be covered by `textChange` - not sure why folks would need to know about an explicit clear event - just defaulted to fine grained control

Reply:
* the stateful version doesn't really need the event - but i think we do need on the stateless input/base input so that whatever controls the state of the input can clear the value string. does seem right?